packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -117,13 +112,7 @@ export const buildModules = async (
           fileInfo.name
         ),
       },
-      esbuild: {
-        logOverride: {
-          "css-syntax-error": "silent",
-        },
-      },
       build: {
-        chunkSizeWarningLimit: 2000,
```

Comment:
I responded to the last comment that we can keep this since the default is 500. Didn't realize that.